### ISSUE TITLE
Jasmine does not seem to launch

### ISSUE THREAD

**Issue by PierreCBSI** [xolvio/chimp#340]
#####  Expected behaviour
I'm following https://chimp.readme.io/docs/getting-started-jasmine
(have no problem with Mocha / cucumber. I have example_spec.js file in /features/) 

When I launch 
$chimp --jasmine --watch

#####  Actual behaviour
it throws an error (see output below)

##### Steps to reproduce the problem
$chimp --jasmine --watch

#####  Used version
* Chimp: 0.9.4
* Node.js: 4.2.6
* Operation system: OSX

#####  Console / Log Output

```
sudo chimp --jasmine --watch
Password:
[chimp] Watching features with tagged with @dev,@watch,@focus

[chimp] Running...
xyz/features/example_spec.js:3
describe('Chimp Jasmine', function () {
^

ReferenceError: describe is not defined
    at Object.<anonymous> (example_spec.js:1:1)
    at Module._compile (module.js:410:26)
    at loader (/usr/local/lib/node_modules/chimp/node_modules/babel-register/lib/node.js:126:5)
    at Object.require.extensions.(anonymous function) [as .js] (/usr/local/lib/node_modules/chimp/node_modules/babel-register/lib/node.js:136:7)
    at Module.load (module.js:344:32)

    at Function.Module._load (module.js:301:12)
    at Module.require (module.js:354:17)
    at require (internal/module.js:12:17)
    at /usr/local/lib/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/support_code_loader.js:63:29
    at Array.forEach (native)
    at Object.wrapper (/usr/local/lib/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/support_code_loader.js:62:15)
    at Object.initializer (/usr/local/lib/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/support_code_loader.js:24:41)
    at Object.Library (/usr/local/lib/node_modules/chimp/node_modules/cucumber/lib/cucumber/support_code/library.js:122:25)
    at Object.getSupportCodeLibrary (/usr/local/lib/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/support_code_loader.js:10:58)
    at Object.getSupportCodeLibrary (/usr/local/lib/node_modules/chimp/node_modules/cucumber/lib/cucumber/cli/configuration.js:127:32)
    at Object.getSupportCodeLibrary (/usr/local/lib/node_modules/chimp/node_modules/cucumber/lib/cucumber/runtime.js:43:46)
```

**Comment by Sanjo** [comment]
You need to update Chimp. Your current version is very old. The up to date version is "0.33.1". `npm install chimp@^0.33.1 --save-dev`.

**Comment by PierreCBSI** [comment]
I did a local and global of that command but checking on the chimp ver ($chimp -v) I still get 0.9.... Is there an argument I should be using? I assume --save-dev shouldn't mess with the install (even though I never use it) Should I use chimp@latest? 
Also, I installed chimp for the first time with npm install less than a month ago - how come I have such an old version?

**Comment by Sanjo** [comment]
Then you are fine. `chimp -v` returns the version of CucumberJS (which makes not much sense :D). You can see the version in node_modules/chimp/package.json.

**Comment by PierreCBSI** [comment]
Then my issue is still valid, `chimp --jasmine --watch ` throws the error noted earlier. 

I can't find the package.json file, I'm in /node_modules/chimp/node_modules/ but no file... am I in the right place?

**Comment by samhatoum** [comment]
try removing all chimp installs from your system - global and local - then do a fresh chimp install

**Comment by PierreCBSI** [comment]
local uninstall: `npm uninstall chimp`
`npm WARN uninstall not installed in /Users/pierre/node_modules/chimp/node_modules: "chimp"`

global uninstall: `sudo npm uninstall chimp -g`
`unbuild chimp@0.33.1`

I'm getting chimp@0.33.1 installed via `npm install chimp -g`
```
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated graceful-fs@2.0.3: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.
```

I still can't find the package.json file in /node_modules/ or /node_modules/chimp/node_modules/.....

(and the same error occurs)

**Comment by PierreCBSI** [comment]
I guess I tried the wrong local uninstall from the first `warn`, I'll try re-uninstalling from /usr/local/lib/node_modules/....

**Comment by PierreCBSI** [comment]
I have gotten rid of local and global chimp -`chimp` cmd no longer recognized. 
From there trying out a chimp global install: `npm install -g chimp`
(output) 
```
> fibers@1.0.10 install /Users/pjohnson/.npm-packages/lib/node_modules/chimp/node_modules/fibers
> node build.js || nodejs build.js

`darwin-x64-v8-4.5` exists; testing
Binary is fine; exiting
npm WARN deprecated jade@0.26.3: Jade has been renamed to pug, please install the latest version of pug instead of jade
npm WARN deprecated graceful-fs@2.0.3: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.

> fsevents@0.3.8 install /Users/pjohnson/.npm-packages/lib/node_modules/chimp/node_modules/chokidar/node_modules/fsevents
> node-gyp rebuild

  SOLINK_MODULE(target) Release/.node
  CXX(target) Release/obj.target/fse/fsevents.o
  SOLINK_MODULE(target) Release/fse.node
npm WARN deprecated graceful-fs@3.0.8: graceful-fs version 3 and before will fail on newer node releases. Please update to graceful-fs@^4.0.0 as soon as possible.

/Users/pjohnson/.npm-packages/bin/chimp -> /Users/pjohnson/.npm-packages/lib/node_modules/chimp/bin/chimp.js
chimp@0.33.1 /Users/pjohnson/.npm-packages/lib/node_modules/chimp
├── chai-as-promised@5.1.0
├── freeport@1.0.5
├── deep-extend@0.4.1
├── child-process-debug@0.0.7
├── underscore@1.8.3
├── progress@1.1.8
├── async@0.9.2
├── chimp-widgets@0.2.0
├── saucelabs@0.1.1
├── colors@1.1.2
├── minimist@1.2.0
├── exit@0.1.2
├── loglevel@1.4.0
├── bluebird@2.10.2
├── commander@2.9.0 (graceful-readlink@1.0.1)
├── fibers@1.0.10
├── chai@3.0.0 (assertion-error@1.0.1, type-detect@1.0.0, deep-eql@0.1.3)
├── xolvio-jasmine-expect@1.0.0 (jasmine-core@2.4.1)
├── glob@6.0.4 (path-is-absolute@1.0.0, inherits@2.0.1, once@1.3.3, inflight@1.0.4, minimatch@3.0.0)
├── requestretry@1.5.0 (fg-lodash@0.0.2)
├── fs-extra@0.24.0 (path-is-absolute@1.0.0, jsonfile@2.3.0, graceful-fs@4.1.3, rimraf@2.5.2)
├── jasmine@2.4.1 (jasmine-core@2.4.1, glob@3.2.11)
├── xolvio-ddp@0.12.0 (ddp-underscore-patched@0.8.1-2, ddp-ejson@0.8.1-3, faye-websocket@0.9.4, request@2.53.0)
├── mocha@2.3.4 (escape-string-regexp@1.0.2, commander@2.3.0, diff@1.4.0, supports-color@1.2.0, growl@1.8.1, debug@2.2.0, glob@3.2.3, mkdirp@0.5.0, jade@0.26.3)
├── request@2.67.0 (aws-sign2@0.6.0, forever-agent@0.6.1, tunnel-agent@0.4.2, oauth-sign@0.8.1, caseless@0.11.0, is-typedarray@1.0.0, stringstream@0.0.5, isstream@0.1.2, json-stringify-safe@5.0.1, extend@3.0.0, tough-cookie@2.2.2, node-uuid@1.4.7, qs@5.2.0, form-data@1.0.0-rc4, c
ombined-stream@1.0.5, mime-types@2.1.10, bl@1.0.3, hawk@3.1.3, http-signature@1.1.1, har-validator@2.0.6)
├── chokidar@1.0.6 (is-glob@1.1.3, arrify@1.0.1, path-is-absolute@1.0.0, async-each@0.1.6, is-binary-path@1.0.1, glob-parent@1.3.0, readdirp@1.4.0, fsevents@0.3.8, anymatch@1.3.0)
├── selenium-standalone@4.9.1 (commander@2.6.0, urijs@1.16.1, minimist@1.1.0, async@1.2.1, mkdirp@0.5.0, which@1.1.1, request@2.51.0, unzip@0.1.11, lodash@3.9.3)
├── hapi@8.8.0 (statehood@2.1.1, items@1.1.0, peekaboo@1.0.0, ammo@1.0.0, accept@1.0.0, catbox-memory@1.1.1, boom@2.7.2, cryptiles@2.0.4, kilt@1.1.1, shot@1.5.3, topo@1.0.2, call@2.0.1, iron@2.1.2, wreck@6.0.0, qs@4.0.0, catbox@4.3.0, hoek@2.14.0, mimos@2.0.2, h2o2@4.0.1, inert@2
.1.5, vision@2.0.1, subtext@1.1.1, joi@6.4.1, heavy@3.0.0)
├── babel-plugin-transform-runtime@6.7.5 (babel-runtime@5.8.38)
├── cucumber@0.9.4 (callsite@1.0.0, stack-chain@1.3.7, cli-table@0.3.1, camel-case@1.2.2, cucumber-html@0.2.3, meteor-promise@0.5.1, gherkin@2.12.2, duration@0.2.0, lodash@4.11.1)
├── babel-core@6.7.6 (slash@1.0.0, babel-messages@6.7.2, babel-template@6.7.0, path-exists@1.0.0, shebang-regex@1.0.0, path-is-absolute@1.0.0, babel-helpers@6.6.0, private@0.1.6, babylon@6.7.0, convert-source-map@1.2.0, source-map@0.5.3, debug@2.2.0, minimatch@2.0.10, babel-types
@6.7.2, babel-code-frame@6.7.5, babel-generator@6.7.5, babel-traverse@6.7.6, json5@0.4.0, lodash@3.10.1, babel-runtime@5.8.38)
├── babel-preset-es2015@6.6.0 (babel-plugin-transform-es2015-typeof-symbol@6.6.0, babel-plugin-transform-es2015-for-of@6.6.0, babel-plugin-transform-es2015-block-scoped-functions@6.6.5, babel-plugin-transform-es2015-template-literals@6.6.5, babel-plugin-check-es2015-constants@6.7
.2, babel-plugin-transform-es2015-literals@6.5.0, babel-plugin-transform-es2015-spread@6.6.5, babel-plugin-transform-es2015-destructuring@6.6.5, babel-plugin-transform-es2015-arrow-functions@6.5.2, babel-plugin-transform-es2015-block-scoping@6.7.1, babel-plugin-transform-es2015-d
uplicate-keys@6.6.4, babel-plugin-transform-es2015-shorthand-properties@6.5.0, babel-plugin-transform-es2015-classes@6.6.5, babel-plugin-transform-es2015-unicode-regex@6.5.0, babel-plugin-transform-es2015-sticky-regex@6.5.0, babel-plugin-transform-es2015-function-name@6.5.0, babe
l-plugin-transform-es2015-object-super@6.6.5, babel-plugin-transform-es2015-modules-commonjs@6.7.4, babel-plugin-transform-es2015-parameters@6.7.0, babel-plugin-transform-es2015-computed-properties@6.6.5, babel-plugin-transform-regenerator@6.6.5)
├── babel-preset-stage-2@6.5.0 (babel-plugin-syntax-trailing-function-commas@6.5.0, babel-plugin-transform-object-rest-spread@6.6.5, babel-preset-stage-3@6.5.0)
├── phantomjs-bin@1.0.1
├── babel-register@6.7.2 (path-exists@1.0.0, home-or-tmp@1.0.0, mkdirp@0.5.1, source-map-support@0.2.10, lodash@3.10.1, babel-runtime@5.8.38, core-js@2.2.2)
├── babel-polyfill@6.7.4 (babel-regenerator-runtime@6.5.0, babel-runtime@5.8.38, core-js@2.2.2)
├── babel-runtime@6.6.1 (core-js@2.2.2)
└── xolvio-sync-webdriverio@5.0.0-rc.3 (xolvio-fiber-utils@2.0.3, meteor-promise@0.4.8, wdio-sync@0.4.9, webdriverio@4.0.5)
```
(closing/restarting terminal) 

```
$ chimp -v
bash: chimp: command not found
```

I'm surprised the global instance does not allow me to launch it from the commandline: What am I missing from there? I'm trying to locate the binary -  but the directory it links to (/Users/pjohnson/.npm-packages/lib/node_modules/chimp/bin/chimp.js?) doesn't recognize the command either

```
(actually the only thing in the file is the following)
module.exports = {
  // - - - - CHIMP - - - -
  watch: false,
  watchTags: '@focus',
  offline: false,

  // - - - - CUCUMBER - - - -
  path: './features',

  // '- - - - DEBUGGING  - - - -
  log: 'info',
  debug: false,
  seleniumDebug: false,
  webdriverLogLevel: false,
  // debugBrkCucumber: 5858,
};
```
what am I missing? Do I need to force a local install to have `chimp` cmd recognized?

**Comment by PierreCBSI** [comment]
I've solved my struggle. It seemed it had to do mostly with File Permissions. 
npm for some reasons installed the modules in unexpected folder `/Users/pjohnson/.npm-packages/`
which was not in my $PATH. 
I modified my `.profile` file in `~/ `and added `export PATH=/Users/pjohnson/.npm-packages/bin:$PATH`
to enable the dir. (then reloaded it via `source ~/.profile` for it to take effect.)

From there was able to launch chimp globally, ran into some permission issues - setting myself as the owner of the problematic directories solved the issues. 

`chimp --jasmine --watch` now works properly !
